spec/cobrato/entities/charge_account_spec.rb

```diff
@@ -15,11 +15,14 @@
       registered_charges: true,
       agreement_number: 4576361,
       initial_remittance_number: 1,
-      remittance_cnab_pattern: 240
+      remittance_cnab_pattern: 240,
+      interest_amount_per_month: 1.02,
+      mulct_type: 'percentage',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.